akka-docs/rst/java/stream/stream-cookbook.rst

```diff
@@ -102,6 +102,17 @@ The :class:`Framing` helper class contains a convenience method to parse message
 
 .. includecode:: ../code/docs/stream/javadsl/cookbook/RecipeParseLines.java#parse-lines
 
+Dealing with compressed data streams
+------------------------------------
+
+**Situation:** A gzipped stream of bytes is given as a stream of ``ByteStrings``, for example from a ``FileIO`` source.
```

Comment:
There are a bunch of other occurrences of ``ByteStrings`` in the cookbook, check:

```
git grep '`ByteStrings`'
```

Would perhaps make sense to fix them all in one go. Another PR or do as a rider here?